### PR TITLE
Feature/display err messages

### PR DESCRIPTION
**Pull request by alexwelch**

### CODE REVIEW

File: test/spec/directives/document-line-edit.js [hunk 1]
```diff
@@ -719,6 +719,106 @@ describe('Directive: documentLineEdit', function () {
...
+          element = compile('<document-line-edit line="line"></document-line-edit>')(scope);
+          scope.$digest();
+        });
+
+        it('joins the errors together', function() {
+          expect(element.isolateScope().errorMessages()).toEqual('');
+        });
+      });
+
+      describe('when a line has both', function() {
```
Comment (alexwelch) [inline test/spec/directives/document-line-edit.js]: hey @dharmamike - I wanna make sure and go over this logic with you... I took an educated guess...

File: app/scripts/directives/document-line-edit.js [hunk 2]
```diff
@@ -149,13 +151,21 @@
...
+        };
+
         scope.getHelpTextForKey = function() {
           var helpText = '';
           if (!lodash.isEmpty($rootScope.keysHelpText)) {
             var node = lodash.find($rootScope.keysHelpText, function(l) {
               return (scope.line.name in l);
             });
-            helpText = node ? node[scope.line.name] : 'Key is invalid.';
+            helpText = node ? node[scope.line.name] : '';
```
Comment (alexwelch) [inline app/scripts/directives/document-line-edit.js]: @rupakg - I'm wondering if we can get rid of all the client side key checking logic here... That may be a good technical debt chore for you to bite off.

File: test/spec/directives/document-line-edit.js [hunk 3]
```diff
@@ -719,6 +719,106 @@ describe('Directive: documentLineEdit', function () {
...
+        beforeEach(function () {
+          scope.line = {name: 'image', value: 'foo:bar', error: ['messed', 'up'], subErrors: {1: ['busted', 'wrong']}};
```
Comment (alexwelch) [inline test/spec/directives/document-line-edit.js]: should be ```errors not error```

**Comment by rupakg** [comment]
@alexwelch lets do a walkthru.

**Comment by dharmamike** [comment]
:+1:

**Comment by rupakg** [comment]
:+1:

**Pull request merged**